### PR TITLE
delete unit_env

### PR DESCRIPTION
**Pull request by TinyuZhao**
Change Name to M5Unit-ENV

### CODE REVIEW

**Review by per1234 — APPROVED** [review]

**Comment by per1234** [comment]
Hello @Tinyu-Zhao. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by umbynos** [comment]
Solved in https://github.com/arduino/library-registry/pull/1576#issuecomment-1178737746

**Pull request merged**
